decidim-collaborative_texts/app/permissions/decidim/collaborative_texts/permissions.rb

```diff
@@ -18,6 +25,15 @@ def permissions
       def document
         @document ||= context.fetch(:document, nil) || context.fetch(:resource, nil)
       end
+
+      def space_allows_admin_access?
+        Decidim.participatory_space_manifests.any? do |manifest|
+          Decidim.find_participatory_space_manifest(manifest.name)
+                 .participatory_spaces.call(user.organization)&.any? do |space|
+            space.admins.exists?(id: user.id)
+          end
+        end
+      end
```

Comment:
I see the rollout permission is used on the update action where you're passing the document itself. Maybe we can simplify like: 

```suggestion
      def space_allows_admin_access?
        return false unless document.participatory_space.respond_to?(:admins)
        
        document.participatory_space.admins.exists?(id: user.id)
      end
```

Reply:
good point